dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -232,7 +232,7 @@ object DLCOracle {
     val key =
       WalletStorage.getPrivateKeyFromDisk(conf.seedPath,
                                           SegWitMainNetPriv,
-                                          password,
+                                          Some(password),
```

Comment:
So we are requiring a password here? It doesn't seem we modified `passwordOpt: Option[AesPassword]` as a parameter to this